### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by sirkitree** [Lullabot/lb-ebook-poc#25]
The task here is to do an accessibility review of https://ebooks.lullabot.com

Here you will find some sample ebooks that are intended to replace our PDF ebooks on the site currently. New books will use this general framework and format. It's been requested that we do an accessibility audit but we're not sure where to start with that.

Whomever picks this up, it would be great to get:
1. a general approach on how we should do this
2. the actual audit
3. the tasks that we should do to reach better compliance
```[tasklist]
### Tasks
- [ ] #37
- [ ] #38
- [ ] https://github.com/Lullabot/lb-ebook-poc/issues/39
- [ ] https://github.com/Lullabot/lb-ebook-poc/issues/41
- [ ] https://github.com/Lullabot/lb-ebook-poc/issues/42
- [ ] https://github.com/Lullabot/lb-ebook-poc/issues/44
```

**Comment by mrdavidburns** [comment]
Added https://ebooks.lullabot.com to our DubBot account.
* https://app.dubbot.com/sites/6696bbc51feb2d000182214f

**Comment by sirkitree** [comment]
thanks for offering to help @claireristow!

I've added a few things that were found in DubBot as tasks. Feel free to add more!

**Comment by mrdavidburns** [comment]
I had DubBot rescan the site. Score is now a 95%. 

https://app.dubbot.com/sites/6696bbc51feb2d000182214f
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/e1693727-8cf2-4817-bb2a-62d52af5e386">

Just 1 more issue.
<img width="1707" alt="image" src="https://github.com/user-attachments/assets/825a6a33-35de-4428-bbf8-3c2bdc4d234f">

```
Link does not have readable (discernible) link text | 2.4.4 Link Purpose (In Context) (WCAG 2.1 Level A)4.1.2 Name, Role, Value (WCAG 2.1 Level A)1194.22 (a) Text equivalent for non-text elements (Section 508)
-- | --
```

**Comment by sirkitree** [comment]
Thanks, addressed this in #46
